packages/build/src/log/messages/install.js

```diff
@@ -1,20 +1,13 @@
 import { isRuntime } from '../../utils/runtime.js'
 import { log, logArray, logSubHeader } from '../logger.js'
 
-export const logInstallMissingPlugins = function (logs, packages) {
-  const runtimes = packages.filter((pkg) => isRuntime(pkg))
```

Comment:
This was always empty, because `isRuntime` expects an object with a `packageName` property whereas the `packages` parameter was an array of strings. I've changed the function to receive both.